diff -u arpwatch-2.1a4/Makefile.in arpwatch-2.1a4.new/Makefile.in
--- arpwatch-2.1a4/Makefile.in	Wed Jul 29 07:16:45 1998
+++ arpwatch-2.1a4.new/Makefile.in	Wed Feb 17 00:54:19 1999
@@ -43,7 +43,7 @@
 
 CC = @CC@
 CCOPT = @V_CCOPT@
-INCLS = -I. @V_INCLS@
+INCLS = -I. @V_INCLS@ -I/usr/include/pcap
 DEFS = -DDEBUG @DEFS@ -DARPDIR=\"$(ARPDIR)\" -DPATH_SENDMAIL=\"$(SENDMAIL)\"
 
 # Standard CFLAGS
@@ -109,13 +109,13 @@
 	$(CC) $(CFLAGS) -o $@ zap.o intoa.o -lutil
 
 install: force
-	$(INSTALL) -m 555 -o bin -g bin arpwatch $(DESTDIR)$(BINDEST)
-	$(INSTALL) -m 555 -o bin -g bin arpsnmp $(DESTDIR)$(BINDEST)
+	$(INSTALL) -s arpwatch $(DESTDIR)$(BINDEST)
+	$(INSTALL) -s arpsnmp $(DESTDIR)$(BINDEST)
 
 install-man: force
-	$(INSTALL) -m 444 -o bin -g bin $(srcdir)/arpwatch.8 \
+	$(INSTALL) $(srcdir)/arpwatch.8 \
 	    $(DESTDIR)$(MANDEST)/man8
-	$(INSTALL) -m 444 -o bin -g bin $(srcdir)/arpsnmp.8 \
+	$(INSTALL) $(srcdir)/arpsnmp.8 \
 	    $(DESTDIR)$(MANDEST)/man8
 
 lint:	$(GENSRC) force
--- arpwatch/configure.in.org	2008-11-01 14:11:41.708042556 +0100
+++ arpwatch/configure.in	2008-11-01 14:11:51.594708133 +0100
@@ -47,7 +47,7 @@
 	;;
 esac
 
-AC_LBL_CHECK_TYPE
+dnl AC_LBL_CHECK_TYPE
 
 AC_LBL_DEVEL(V_CCOPT)
 
