Index: arpwatch/bihourly
diff -u arpwatch/bihourly:1.1.1.1 arpwatch/bihourly:1.1.1.1.18.1
--- arpwatch/bihourly.sh:1.1.1.1	Tue Apr 17 13:31:36 2001
+++ arpwatch/bihourly.sh	Wed Aug 11 21:54:18 2004
@@ -10,8 +10,8 @@
 #
 list="`cat list`"
 cname="`cat cname`"
-temp1=/tmp/bihourly.1.$$
-temp2=/tmp/bihourly.2.$$
-d=/tmp/errs
+temp1=`mktemp`
+temp2=`mktemp`
+d=`mktemp -d`
 
 # imperfect hack
Index: arpwatch/mkdep
diff -u arpwatch/mkdep:1.1.1.1 arpwatch/mkdep:1.1.1.1.18.1
--- arpwatch/mkdep:1.1.1.1	Tue Apr 17 13:31:37 2001
+++ arpwatch/mkdep	Wed Aug 11 21:54:18 2004
@@ -51,7 +51,7 @@
 	exit 1
 fi
 
-TMP=/tmp/mkdep$$
+TMP=`mktemp -p mkdep -s .tmp`
 
 trap 'rm -f $TMP ; exit 1' 1 2 3 13 15
 
