diff -urNp -x '*.orig' arpwatch-2.1a15.org/Makefile.in arpwatch-2.1a15/Makefile.in
--- arpwatch-2.1a15.org/Makefile.in	2021-05-09 14:42:25.080236575 +0200
+++ arpwatch-2.1a15/Makefile.in	2021-05-09 14:42:25.160240737 +0200
@@ -45,7 +45,7 @@ VPATH = @srcdir@
 CC = @CC@
 PROG = arpwatch
 CCOPT = @V_CCOPT@
-INCLS = -I. @V_INCLS@
+INCLS = -I. @V_INCLS@ -I/usr/include/pcap
 DEFS = -DDEBUG @DEFS@ -DARPDIR=\"$(ARPDIR)\" -DPATH_SENDMAIL=\"$(SENDMAIL)\" \
 	-DETHERCODES=\"$(ETHERCODES)\"
 
@@ -113,13 +113,13 @@ zap: zap.o intoa.o
 	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ zap.o intoa.o -lutil
 
 install: force
-	$(INSTALL) -m 555 -o bin -g bin arpwatch $(DESTDIR)$(BINDEST)
-	$(INSTALL) -m 555 -o bin -g bin arpsnmp $(DESTDIR)$(BINDEST)
+	$(INSTALL) -s arpwatch $(DESTDIR)$(BINDEST)
+	$(INSTALL) -s arpsnmp $(DESTDIR)$(BINDEST)
 
 install-man: force
-	$(INSTALL) -m 444 -o bin -g bin $(srcdir)/arpwatch.8 \
+	$(INSTALL) $(srcdir)/arpwatch.8 \
 	    $(DESTDIR)$(MANDEST)/man8
-	$(INSTALL) -m 444 -o bin -g bin $(srcdir)/arpsnmp.8 \
+	$(INSTALL) $(srcdir)/arpsnmp.8 \
 	    $(DESTDIR)$(MANDEST)/man8
 
 lint:	$(GENSRC) force
diff -urNp -x '*.orig' arpwatch-2.1a15.org/configure.in arpwatch-2.1a15/configure.in
--- arpwatch-2.1a15.org/configure.in	2021-05-09 14:42:25.090237095 +0200
+++ arpwatch-2.1a15/configure.in	2021-05-09 14:42:25.160240737 +0200
@@ -49,7 +49,7 @@ osf3*)
 	;;
 esac
 
-AC_LBL_CHECK_TYPE
+dnl AC_LBL_CHECK_TYPE
 
 AC_LBL_DEVEL(V_CCOPT)
 
